cob_default_robot_config/cob4-1/command_gui_buttons.yaml

```diff
@@ -4,45 +4,13 @@
 #  buttons: [[button_name1,function_name1,parameter_name2,
 #    [button_name2,function_name2,parameter_name2]]
 
-group1:
+group10:
```

Comment:
For cob4-2, there are more than 10 columns for knoeppkes.
According to @ipa-nhg, the ordering was faulty when using single-digit groups (0-9) together with dual-digit groups (10-99)...
The values have been copied from the current cob4-2 config...